frontend/javascripts/types/schemas/datasource.schema.ts

```diff
@@ -64,36 +65,44 @@ export default {
       },
       required: ["dataFormat", "boundingBox", "wkwResolutions"],
     },
-    "types::DataLayerKnossosPartial": {
-      title: "DataLayerKnossos",
+    "types::DataLayerZarrPartial": {
+      title: "DataLayerZarr",
       type: "object",
       properties: {
         dataFormat: {
-          const: "knossos",
+          const: "zarr",
         },
-        sections: {
+        boundingBox: {
+          $ref: "#/definitions/types::BoundingBox",
+        },
+        mags: {
           type: "array",
           items: {
             type: "object",
             properties: {
-              name: {
-                type: "string",
+              mag: {
+                anyOf: [
+                  {
+                    type: "number",
+                  },
+                  {
+                    $ref: "#/definitions/types::Vector3",
+                  },
+                ],
               },
-              resolutions: {
-                type: "array",
-                items: {
-                  type: "number",
-                },
+              path: {
+                type: "string",
               },
-              boundingBox: {
-                $ref: "#/definitions/types::BoundingBox",
+              axisOrder: {
+                type: "object",
+                additionalProperties: { type: "number" },
               },
             },
-            required: ["name", "resolutions", "boundingBox"],
+            required: ["mag", "path"],
```

Comment:
Note that `path` in the mag object is not required. Another optional field within the mag object is `credentials`.      example:
```    "credentials": {
            "user": "asdf",
            "password": "asdf"
          }
```

Reply:
thanks, will update that! out of curiosity: where would wk fetch the data from if the path is not provided?

Reply:
it will look on the local datastore disc, relative to the dataset dir in `layer_name/2-2-1` (short form for scalar mags, i.e. `layer_name/1`)

Reply:
ah, makes sense :+1: thanks!